components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/GroupDAO.java

```diff
@@ -423,6 +423,7 @@ public void updateRoleName(int tenantId, String oldRoleName, String newRoleName)
 
         // Resolve sql query for filtering.
         if (StringUtils.isNotEmpty(domainName)) {
+
```

Comment:
No need to have a new line.

Reply:
Fixed: 7f95a33846b4f8b080987cf395121938ee0c6ceb